person.rb

```diff
@@ -20,6 +22,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person(self)
+  end
+
```

Comment:
- [ ] It would be nice if you make your add_rental method as shown below to make your code work well and to be consistent with the method which you have created in the book class 👍

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
- [ ] Please address what previous reviewer suggested